Sources/Orbit/Support/Components/TimelineStep.swift

```diff
@@ -1,37 +1,52 @@
 import SwiftUI
 
+public extension VerticalAlignment {
+    enum TimelineStepAlignment : AlignmentID {
+        public static func defaultValue(in dimensions: ViewDimensions) -> CGFloat {
+            return dimensions[VerticalAlignment.top]
+        }
+
+        public static func computeFirstLineCenter(in dimensions: ViewDimensions) -> CGFloat {
+            (dimensions.height - (dimensions[.lastTextBaseline] - dimensions[.firstTextBaseline])) / 2
+        }
+    }
+
+    static let timelineStepAlignment = VerticalAlignment(TimelineStepAlignment.self)
+}
+
 /// One item of a Timeline.
 ///
 /// - Related components
 ///   - ``Timeline``
 ///
 /// - Note: [Orbit definition](https://orbit.kiwi/components/progress-indicators/timeline/)
-public struct TimelineStep: View {
+public struct TimelineStep<Header: View, Content: View>: View {
 
-    public static let indicatorDiameter: CGFloat = Icon.Size.large.value
+    @Environment(\.sizeCategory) var sizeCategory
 
-    let label: String
-    let sublabel: String
-    let style: Style
-    let content: String
+    let style: TimelineStepStyle
+    let isIconFirstTextLineCentered: Bool
```

Comment:
Lets use the native `VerticalAlignment` with a default value of probably `.firstLineBaseline`. Don't we always want this alignment btw?

Reply:
I don't think so. `firstLineBaseline` is not a center of the text, so the icon looks misaligned. 
I'm not sure that we can easily switch between `timelineStepAlignment` and `native VerticalAlignment` based on flag.

I've left an option to use `.top` aligned but in current examples, it's not used, all of those previews are using custom alignment.